content/en/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+In robotics and automation, a _control loop_ is
+a non-terminating loop that regulates the state of a system.
+
+Here is one example of a control loop: a thermostat in a room.
+
+When you set the temperature, that's telling the thermostat
+about your *desired state*. The actual room temperature is the
+*current state*. The thermostat acts to bring the current state
+closer to the desired state, by turning equipment on or off.
+
+In Kubernetes, a _controller_ is a control loop that watches the
+state of your
+{{< glossary_tooltip term_id="cluster" text="cluster">}}, then
+makes or requests changes. Each controller tries to move the current
+cluster state closer to the desired state.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Controller pattern
+
+A controller tracks at least one Kubernetes resource type.
+These [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/)
+and their associated data are the desired state, so the
+controller(s) for that resource are responsible for making the current
+state come closer to that desired state.
+
+The controller might carry the action out itself; more commonly, in Kubernetes,
+a controller will send messages to the
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} that have
+useful side effects.
+
+### Control via API server
+
+The controller for Job is an example of a built-in controller that
+makes all of its changes by interacting with your cluster's API server.
+
+{{< glossary_tooltip term_id="job" >}} is a Kubernetes resource that
+runs a {{< glossary_tooltip term_id="pod" >}}, or perhaps several Pods,
+to carry out a task and then stop.
+
+(Once [scheduled](/docs/concepts/scheduling/), Pod objects become part
+of the desired state for a kubelet).
+
+When the Job controller sees a new task it makes sure that, somewhere
+in your cluster, the kubelets on a set of Nodes are running the right
+number of Pods to get the work done.
+The Job controller does not run any Pods or containers
+itself. Instead, the Job controller tells the API server to create or remove
+Pod [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/).
+Other components in the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}
+act on the new information (there are new Pods to schedule and run),
+and eventually the work is done.
+
+After you create a new Job, the desired state is for that Job to
+be completed. The Job controller makes the current state be nearer
```

Comment:
Does this include running the Job (create, start (run), stop or completed)?
The Job controller makes the current state [of the Job?] closer to the desired state by ... creating Pods ...